### ISSUE TITLE
please provide 32&64 bit version

### ISSUE THREAD

**Issue by h0nIg** [YvesR/mod_authn_ntlm#13]
Hey,

after the last pull request, the created so file is only 64 bit. Could you please provide a 32 and a 64 version, since apache is complaining about the .so file

```
httpd.exe: Syntax error on line 116 of C:/Apache24/conf/httpd.conf: Cannot load modules/mod_authn_ntlm.so into server: %1 ist keine zul\xe4ssige Win32-Anwendung.
```

@YvesR @lwahonen 

thanks!

**Comment by lwahonen** [comment]
Hi,

Did you compile the file yourself? Did you notice the comments on top of the Makefile?

BR,
Will

PS: You probably get that same error message if you're missing the Visual C++ Runtime for the compiler version you used. The version in the bin directory is built with VC2012 IIRC.

**Comment by JBlond** [comment]
There is a 32 bit and a 64 bit version on ApacheHaus.com

**Comment by h0nIg** [comment]
@JBlond the version that is distributed via apachehaus is very old (2013)

**Comment by JBlond** [comment]
Well the VS 2015 version is from Januray this year. 

Why very old? The source code did not change that much since that "release". There are two IE fixes. 

mod_authn_ntlm never had tags or releases on github.Kinda hard to follow. 
I will try to find some time to compile it with VS2012 based on the last available source code.

**Comment by JBlond** [comment]
I did the VC2012 build x86 and x64. They are now in the download page of ApacheHaus

Make sure you install the correct runtime files! http://www.microsoft.com/en-us/download/details.aspx?id=30679

**Comment by JBlond** [comment]
@h0nIg can you close the issue now? Or is there is something to do?
